harvard-lts/mps-embed
# Increasing Test Code Coverage and Fixing GitHub Actions.

**Increasing Test Code Coverage and Fixing GitHub Actions.**
* * *

**JIRA Ticket**: [LTSVIEWER-279)](https://jira.huit.harvard.edu/browse/LTSVIEWER-279)

# What does this Pull Request do?
This increases the overall code coverage in the Jest tests from 67.85% to 93.72%. Additionally it fixes the GitHub Action that runs the test. The Action completes successfully. Finally, I upgraded the Checkout step of the Action to use version3, which upgrades Node to version 16. 

# How should this be tested?

A description of what steps someone could take to:
* Build mps-embed using the `LTSVIEWER-279` branch.
* Next, have your mps-viewer container running.
* Bash into the mps-embed container: `docker exec -it mps-embed bash`
* Run all the tests: `npm run test`
* Confirm that all 41 tests pass. Confirm the code coverage for "All files" is 93.72%. 
* Confirm that the GitHub Action associated with this PR finishes successfully. (https://github.com/harvard-lts/mps-embed/actions/runs/6236404739/job/16927615156?pr=43)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Yes
- integration tests? Yes

# Interested parties
@f8f8ff @enriquediaz
